packages/rolldown/tests/cli/fixtures/config-multiply-output-with-options-hooks/rolldown.config.ts

```diff
@@ -0,0 +1,22 @@
+import { defineConfig } from 'rolldown'
```

Comment:
It seems this behavior is not related to cli, put it under `packages/rolldown/tests/fixtures/plugin` maybe more proper.

Reply:
The multiply output is a cli feature, the js API `build.generate()` is not support it.